roles/edpm_nova/tasks/adoption.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Stop and disable compute services
+  no_log: "{{ edpm_network_config_hide_sensitive_logs | bool }}"
```

Comment:
hm, this variable seems to be intended to the network_config roles. Do we even foresee some sensitive logs here?

Reply:
done, removed